lib/audited/auditor.rb

```diff
@@ -60,8 +60,12 @@ def audited(options = {})
 
         attr_accessor :audit_comment
 
-        has_many :audits, -> { where(service_name: Rails.application.class.parent_name).order(version: :asc) }, as: :auditable, class_name:  
-        Audited.audit_class.audited_class_names << to_s
+        has_many :audits, ->(audited_record) do
+          where(
+            service_name: Rails.application.class.parent_name,
+            created_at: (audited_record.created_at - 1.minute)..(Time.now)
```

Comment:
Just to make sure that first "creation" audit is not filtered out I added `- 1.minute` this could be removed if properly tested but as quick fix I would leave it be